src/renderer/components/monaco-editor/get-editor-height-from-lines-number.injectable.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+import { getInjectable } from "@ogre-tools/injectable";
+
+const getEditorHeightFromLinesCountInjectable = getInjectable({
+  id: "get-editor-height-from-lines-number",
+
+  instantiate: () => {
+    return (linesNumber: number) => {
+      if (typeof linesNumber !== "number") {
+        throw new Error("linesNumber must be a number");
+      }
+
+      if (linesNumber < 10) {
+        return "small";
+      }
+
+      if (linesNumber < 20) {
+        return "medium";
+      }
+
+      return "large";
```

Comment:
Why do we return predefined names instead real numbers? Otherwise these built-in constants locked with css-names (e.g. "small")? And what the benefit then from your injectable when it receives current editor height-number and should return predefined css-class names?